src/content/docs/release-process.md

```diff
@@ -53,6 +53,12 @@ yarn run storybook
 # Test storybook static build
 yarn run build:storybook && yarn run start:storybook-static # test storybook  static export
 
+# Optionnaly audit your page
+# Run the app
+# yarn run dev
```

Comment:
You probably want to audit built app and not dev app (for performances mainly, auditing dev would be only relevant for accessibility)